### PR TITLE
2666 and 2698: Fix ios promotion by upgrading fastlane (and other gems)

### PR DESCRIPTION
**Pull request by steffenkleinle**
### Short description

Fix ios promotion by upgrading fastlane (and other gems)

### Proposed changes

Update all gems in native, android and ios.
Updating fastlane should fix both #2666 and #2698.

### Side effects

None.

### Resolved issues

Fixes: #2666, #2698.

---

### CODE REVIEW

File: native/ios/Gemfile.lock [hunk 1]
```diff
@@ -272,13 +273,14 @@ GEM
       xcpretty (~> 0.2, >= 0.0.7)
 
 PLATFORMS
-  ruby
```
Comment (steffenkleinle) [inline native/ios/Gemfile.lock:275]: Any idea why this only updates for ios but not the android or native? Same for `BUNDLED WITH` below...

**Comment by LeandraH** [comment]
I think the Fastfiles still need to be updated.

File: native/ios/Gemfile.lock (hunk 1 above)
Comment (f1sh1918) [reply native/ios/Gemfile.lock:275]: The platform should be readded!!
As i found out the bundler check the version of the current `Gemfile.lock` and uses this. When updating rubygems you have to remove the `Gemfile.lock` and rereun `bundle install` to get correct version and platforms.
Please do that for `native` & `android` & `ios`

**Review by f1sh1918 — APPROVED** [review]
looks good 👍

**Pull request merged**